llm_demo/app.py

```diff
@@ -180,7 +185,7 @@ async def book_flight(request: Request, params: str = Body(embed=True)):
     request.session["history"].append(
         {"type": "ai", "data": {"content": "I have booked your ticket."}}
     )
-    return response
+    return json.dumps(response)
```

Comment:
should the other endpoints return json as well?

Reply:
Updated the return from retrieval service.